model/src/main/kotlin/DependencyGraph.kt

```diff
@@ -295,7 +295,7 @@ data class RootDependencyIndex(
  * Note: This is by intention no data class. Equality is tested via references and not via the values contained.
  */
 @JsonInclude(JsonInclude.Include.NON_DEFAULT)
-class DependencyReference(
+data class DependencyReference(
```

Comment:
Commit message: Could you please add a permalink to the exact test case? I'm having trouble finding it.

Reply:
I've also re-written the commit message as the thing with the `copy()` was not exactly right.